pages/_document.js

```diff
@@ -26,7 +26,7 @@ export default class MyDocument extends Document {
   render() {
     return (
       <Html lang="en">
-        <Head />
+        <Head></Head>
```

Comment:
since  <Head></Head> doesn't have any content and children, it can be a self closing tag.